test/src/keyboard.dart

```diff
@@ -23,9 +25,20 @@ void runTests() {
   group('Keyboard', () {
     WebDriver driver;
     WebElement textInput;
+    String ctrlCmdKey = '';
 
     setUp(() async {
-      driver = await createTestDriver();
+      if (Platform.isMacOS) {
+        ctrlCmdKey = Keyboard.command;
+      } else {
+        ctrlCmdKey = Keyboard.control;
+      }
+
+      // Note: Firefox used as Chrome + Mac OSX prevents use of control/meta
+      // in chords.
+      // https://code.google.com/p/selenium/issues/detail?id=5919
```

Comment:
Probably better to reference the chromedriver issue:
https://bugs.chromium.org/p/chromedriver/issues/detail?id=30